CONTRIBUTING.md

```diff
@@ -0,0 +1,46 @@
+# Contributing Guidelines
+
+*Pull requests, bug reports, and feature requests are welcomed and highly encouraged!*
+
+### Contents
+
+- [General Guidelines](#📖-general-guidelines)
+- [Questions](#💡-questions)
+- [Opening an Issue](#📥-opening-an-issue)
+- [Feature Requests](#💌-feature-requests)
+- [Submitting Pull Requests](#🔁-submitting-pull-requests)
+
+## 📖 General Guidelines
+
+For everything from pull requests, bug reports, features requests and code reviews the following applies:
+
+**Be respectful of each other!** Abusive language or harassment is not welcome here!
+
+## 💡 Questions
+
+The [The Claw Discord](https://discord.gg/theclaw) or [Whitep4nth3r's twitch chat](https://www.twitch.tv/whitep4nth3r) are the places to be for any questions regarding the code or p4nth3rworld itself. 
+Please don't open issues for questions to not create any clutter.
+
+## 📥 Opening an Issue
+
+If you find any issues with the p4nth3rworld site or p4nth3rworld the game we would love a well-written report!
+
+- **Do not open a duplicate issue!** Search through existing issues to see if your issue has previously been reported. If your issue exists, comment with any additional information you have. You may simply note "I have this problem too", which helps prioritize the most common problems and requests. 
+
+- **Prefer the use of [reactions](https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/)**, not comments, if you simply want to "+1" an existing issue.
```

Comment:
```suggestion
- **The use of [reactions](https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/)** instead of comments is preferred, if you simply want to "+1" an existing issue.
```